src/pyvesync/vesync.py

```diff
@@ -718,15 +719,15 @@ def turn_off(self):
             return False
 
 
-class VeSyncSwitchEU10A(VeSyncSwitch):
```

Comment:
Why take out the EU?  This is the only outlet that supports European Outlets to my knowledge

Reply:
I just changed the class name, as this class is now used for both the EU plug and this new USA model. Both are supported.

Reply:
Is that the new outdoor outlet?  Do you have one of them?

Reply:
It's a new model of the mini US plug (ESW01-USA). This one supports 10A.

I do have this model and tested against it.

Reply:
I got excited, I thought it might have been one of the other newer models I saw in the app.  The ESW01 was the original outlet that this project supported (the 7A switch).  Have you done packet captures?  If so, in the device list what does the devType show?  Did you try to control those outlets with this program before you forked and did it completely not show up or did you receive any errors?

Reply:
```json
{
  "deviceName": "Round",
  "deviceImg": "https://smartapi.vesync.com/v1/app/imgs/icon_7a_wifi_outlet/icon_7a_outlet_80.png",
  "cid": "<cid>",
  "deviceStatus": "off",
  "connectionStatus": "online",
  "connectionType": "wifi",
  "deviceType": "ESW03-USA",
  "type": "wifi-switch",
  "uuid": "<uuid>",
  "configModule": "10AOutletUS",
  "macID": null,
  "mode": null,
  "speed": null,
  "extension": null,
  "currentFirmVersion": null
}
```

Reply:
Wow I've never seen that before.  Thanks for the update